From f33a71e1dbadf045908b8d33464899efba017dda Mon Sep 17 00:00:00 2001
From: Lionel Debieve <lionel.debieve@st.com>
Date: Thu, 10 Nov 2016 14:53:35 +0100
Subject: [PATCH 4/4] Remove unnecessary configuration

Xenomai support need to remove CMA and
Frequency scaling to avoid latency issue

Change-Id: I9661e6e8a53de1b5804872d2c4e31a437d2499c3
Signed-off-by: Lionel Debieve <lionel.debieve@st.com>
---
 arch/arm/configs/multi_v7_defconfig | 14 --------------
 1 file changed, 14 deletions(-)

diff --git a/arch/arm/configs/multi_v7_defconfig b/arch/arm/configs/multi_v7_defconfig
index d641712..cc581a6 100644
--- a/arch/arm/configs/multi_v7_defconfig
+++ b/arch/arm/configs/multi_v7_defconfig
@@ -25,22 +25,10 @@ CONFIG_MCPM=y
 CONFIG_NR_CPUS=16
 CONFIG_AEABI=y
 CONFIG_HIGHMEM=y
-CONFIG_CMA=y
 CONFIG_FORCE_MAX_ZONEORDER=12
 CONFIG_ARM_APPENDED_DTB=y
 CONFIG_ARM_ATAG_DTB_COMPAT=y
 CONFIG_KEXEC=y
-CONFIG_CPU_FREQ=y
-CONFIG_CPU_FREQ_STAT_DETAILS=y
-CONFIG_CPU_FREQ_DEFAULT_GOV_ONDEMAND=y
-CONFIG_CPU_FREQ_GOV_POWERSAVE=y
-CONFIG_CPU_FREQ_GOV_USERSPACE=y
-CONFIG_CPU_FREQ_GOV_CONSERVATIVE=y
-CONFIG_CPUFREQ_DT=y
-CONFIG_ARM_STI_CPUFREQ=y
-CONFIG_QORIQ_CPUFREQ=y
-CONFIG_CPU_IDLE=y
-CONFIG_ARM_CPUIDLE=y
 CONFIG_VFP=y
 CONFIG_NEON=y
 CONFIG_KERNEL_MODE_NEON=y
@@ -73,8 +61,6 @@ CONFIG_RFKILL_INPUT=y
 CONFIG_RFKILL_GPIO=y
 CONFIG_DEVTMPFS=y
 CONFIG_DEVTMPFS_MOUNT=y
-CONFIG_DMA_CMA=y
-CONFIG_CMA_SIZE_MBYTES=64
 CONFIG_MTD=y
 CONFIG_MTD_BLOCK=y
 CONFIG_MTD_CFI=y
-- 
2.7.4

